app/models/quiz_item.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+class QuizItem < Item
+  has_many :quiz_question_choices, class_name: 'QuizQuestionChoice', foreign_key: 'question_id', inverse_of: false, dependent: :nullify
+
+  def edit
+  end
+
+  def view_question_text
```

Comment:
I think I would go for "view_item_text".  You might think that everything on a quiz is a "question," but you can still have checkboxes, where the prompts are structured as statements, not questions.